src/app/static/src/tests/testHelpers.ts

```diff
@@ -68,33 +69,36 @@ export function expectAllMutationsDefined(mutationDefinitions: any, mutationTree
     }
 }
 
-export function expectTranslatedWithStoreType<T extends TranslatableState>(element: Wrapper<any>,
+export async function expectTranslatedWithStoreType<T extends TranslatableState>(element: any,
                                                                            englishText: string,
                                                                            frenchText: string,
                                                                            portugueseText: string,
                                                                            store: Store<T>,
                                                                            attribute?: string) {
+    const value = () => attribute ? element.attributes(attribute) : element.text();
     store.state.language = Language.en;
     registerTranslations(store);
```

Comment:
can you await i18n.init? try await in front of expect

Reply:
Following this, I was wondering why registerTranslations was even needed! I just deleted it from the function and everything works! So now it is just, change the store, await nextTick!